hack/benchmark/time-to-k8s/chart.go

```diff
@@ -92,14 +75,15 @@ func readInCSV(csvPath string, apps map[string]runs) error {
 		values := []float64{}
 
 		// 8-13 contain the run results
```

Comment:
Just update this comment to match the range we're using.

Reply:
addressed